GitUI/Editor/FileViewer.cs

```diff
@@ -406,6 +406,11 @@ public int GetLineFromVisualPosY(int visualPosY)
             return internalFileViewer.GetLineFromVisualPosY(visualPosY);
         }
 
+        public int? InitalLineNumber
+        {
+            set => internalFileViewer.InitalLineNumber = value;
```

Comment:
Set-only properties should be turned into methods.

Reply:
I would turn it into another argument to `ViewFileAsync` (advantage: stateless).

BTW: All calls of `ViewFileAsync` should be awaited or "filed".